src/analysis/special_functions/pow.lean

```diff
@@ -1598,6 +1598,39 @@ lemma filter.tendsto.ennrpow_const {α : Type*} {f : filter α} {m : α → ℝ
   tendsto (λ x, (m x) ^ r) f (𝓝 (a ^ r)) :=
 (ennreal.continuous_rpow_const.tendsto a).comp hm
 
+-- the following lemma is placed here instead of `analysis.specific_limits` since
+-- we need `real.rpow`
```

Comment:
can you also add a docstring saying what the lemma says? (Parsing tsums and ite is not that easy...)